### ISSUE TITLE
New section? ""Exhibition and Display tools"?

### ISSUE THREAD

**Issue by hadro** [IIIF/awesome-iiif#304]
I propose that we add @jpadfield's[ Simple Site project extension](https://jpadfield.github.io/simple-site/IIIF%20viewer.html) to the IIIF Awesome List.

I don't think that's controversial, but it does raise a few questions:

1.    Where should it live? I put it under "Presentation Manifest Tools", but that doesn't feel perfect ... which brings me to:
2.    Should we create a new section that also includes these two tools as well, which are potentially similar?

- https://github.com/minicomp/wax
- https://github.com/pbinkley/postwax

among others.

Maybe a section like "Exhibition and Display tools"? I'd love to hear thoughts on this.

(For the sake of transparency, I created this issue after I already made PR #303, but before we merge that we should discuss here, I think)

**Comment by triplingual** [comment]
I think it also raises a question at another layer of abstraction: What happens when the list starts having significant issues of categorization?

This is prompted by thinking about the relationship of Omeka-related links to the new section. Would they need to move to the new section? Be —ugh— cross-listed/-referenced there? No change? If they moved, they would leave just one link in the "CMS Integration" section, but if they didn't move, there'd be meaningful semantic slippage in the new section title. Or, if "CMS Integration" is abandoned in favor of new categories that accommodate current CMS Integration links and these new ones, we break persistence for people replying on CMS Integration as a stable category. On the nth hand, the Omeka links point to plugins, not to the tool itself. I just know that as someone who likes both Wax and Omeka, I'd expect links to one to be pretty close to links to the other.

**Comment by jpadfield** [comment]
I just had a thought :-) 

If you can define a simple json file, or a folder of individual json files (one per project) that defines the awesome examples
categories:
   - Title, comment, keyword
projects:
   - Title, Link, Thumbnail/logo, Comment and then a list of Category keywords

I could use the [simple-site system](https://github.com/jpadfield/simple-site/) to automatically create a set of webpages presenting them all based on the defined categories - projects could be automatically listed in multiple categorise as required, but still only defined once.

Editing the json file(s), which list(s) the projects, adding a new project or adding a category - would force the webpages to be rebuilt as required.

Basically a database of awesome projects without the database?

Would that make it easier?

Oh and thanks for adding the work @hadro :-)

**Comment by jpadfield** [comment]
It would be good to add a last checked date to each one as well, as some may stop working and some may become compromised. 

As an example: http://digirati-co-uk.github.io/iiif-gallery/src/ does not seem to be happy, various CORS errors, and it was hard to see but some of the examples images in the maps example may well be wrong?

**Comment by tomcramer** [comment]
Current thread on iiif-discuss today would support adding an "exhibits-y" category. To Josh's first list we might add Omeka(-S?) and Spotlight.
